package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screener-runner",
-  "version": "0.11.8",
+  "version": "0.11.9",
```

Comment:
No need to update the version - this will get auto-incremented during publish

Reply:
Backed this change